test/e2e_node/quota_lsci_test.go

```diff
@@ -152,7 +157,21 @@ func diskConcealingPod(name string, diskConsumedMB int, volumeSource *v1.VolumeS
 
 // Don't bother returning an error; if something goes wrong,
 // simply treat it as "no".
-func supportsQuotas(dir string) bool {
-	supportsQuota, err := fsquota.SupportsQuotas(mount.New(""), dir)
+func supportsQuotas(dir string, userNamespaceDisabled bool) bool {
+	supportsQuota, err := fsquota.SupportsQuotas(mount.New(""), dir, userNamespaceDisabled)
 	return supportsQuota && err == nil
 }
+
+func supportsUserNS(ctx context.Context, f *framework.Framework) bool {
```

Comment:
hm instead of also adding here maybe this should just be fully rebased on https://github.com/kubernetes/kubernetes/pull/123303/files#diff-3e3f9fb1d6e43deaeeb5420d72c03fea7a0c33da61e5baa0a652e1681b9acb34R125 so we don't get weird merge issues with duplicated function signatures..

Reply:
or, factor this added function out into a separate commit that has the title "WIP: drop when https://github.com/kubernetes/kubernetes/pull/123303 merges"

Reply:
Added it as a separate commit